sync/upstream.py

```diff
@@ -967,5 +967,8 @@ def update_pr(git_gecko, git_wpt, sync, action, merge_sha=None, base_sha=None):
             if sync.status not in ("complete", "wpt-merged"):
                 env.bz.comment(sync.bug, "Upstream PR merged")
                 sync.finish("wpt-merged")
+            elif (sync.status == "wpt-merged" and
+                    merged_by == env.config["web-platform-tests"]["github"]["user"]):
```

Comment:
Incorrect indent.

Reply:
The linter seemed to complain about a shallower indent, will try again.